seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/FlinkCommandArgs.java

```diff
@@ -20,8 +20,14 @@
 import org.apache.seatunnel.common.config.DeployMode;
 import org.apache.seatunnel.config.EngineType;
 
+import com.beust.jcommander.Parameter;
+
 public class FlinkCommandArgs extends AbstractCommandArgs {
 
+    @Parameter(names = {"-r", "--run-mode"},
+        description = "job run mode, run or run-application")
+    private String runMode;
```

Comment:
```suggestion
   @Parameter(names = {"-r", "--run-mode"},
        description = "job run mode, run or run-application",
         required = true)
    private String runMode = "run";
```

Reply:
@ruanwenjun  I think optional will be a better user experience, as we already set the default value.

What do you thins?